app/js/components/answers/Index.react.js

```diff
@@ -46,12 +46,27 @@ class AllAnswers extends React.Component {
      Common.initTinyMceContent('.answer');
    }
 
+   sortAnswers(answers_obj){
```

Comment:
:question: Can this be refactored to use Object.keys and in turn filter it to allow for more succint code? to aid with unitesting